frontend/awx/resources/projects/hooks/useSelectProject.tsx

```diff
@@ -1,25 +1,38 @@
+import { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useSelectDialog } from '../../../../../framework';
 import { Project } from '../../../interfaces/Project';
 import { useAwxView } from '../../../useAwxView';
 import { useProjectsColumns, useProjectsFilters } from '../Projects';
 
-export function useSelectProject() {
+export function useSelectProject(isLookup: boolean) {
   const { t } = useTranslation();
   const toolbarFilters = useProjectsFilters();
   const tableColumns = useProjectsColumns({ disableLinks: true });
+  const columns = useMemo(
+    () =>
+      isLookup
+        ? tableColumns.filter((item) =>
+            ['Name', 'Status', 'Type', 'Organization'].includes(item.header)
```

Comment:
this will break with translations

Reply:
We mentioned during our call on Friday to let this go in and to file a follow up issue to address lookup columns across the board.  Are you still on board with that plan?  If so, I'm happy to file an issue for it.

Reply:
yeah, i think we can add a new prop to columns and have the underlying component filter the columns based on that